src/styles/article-footnotes.css

```
@@ -0,0 +1,188 @@
+  gap: 0.5rem;
+  margin: 0 0 1rem;
+  padding: 0;
+  overflow: visible;
+  border: 0;
+  color: var(--foreground);
+  font-size: var(--font-size-title3);
+  font-weight: 600;
+  line-height: var(--lh-tight);
+  letter-spacing: 0;
+  white-space: normal;
+  clip: auto;
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)